src/plugins/util/mobile-signal-strength.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**
+ * List of possible network signal strengths to send to the VM.
+ * Used only for android versions greater or equal to 8.
+ */
+module.exports = [{
+    id: 4,
+    name: 'great',
+    label: 'Great',
```

Comment:
Use i18n translation

Reply:
Will be done later, as this is a bit out of scope